crates/rattler-bin/src/commands/create.rs

```diff
@@ -367,7 +367,7 @@ async fn install_package_to_environment(
     // Create the conda-meta directory if it doesnt exist yet.
     let target_prefix = target_prefix.to_path_buf();
     match tokio::task::spawn_blocking(move || {
-        let conda_meta_path = target_prefix.join("conda-meta");
+        let conda_meta_path = target_prefix.join("prefix");
```

Comment:
hmm, I wonder if we should keep that at `conda-meta` to not throw off micromamba or conda tools and others

Reply:
woops! thats a mistake!